Kitodo-DataManagement/src/main/resources/db/migration/V2_137__Use_lowercase_table_names.sql

```diff
@@ -0,0 +1,65 @@
+--
+-- (c) Kitodo. Key to digital objects e. V. <contact@kitodo.org>
+--
+-- This file is part of the Kitodo project.
+--
+-- It is licensed under GNU General Public License version 3 or later.
+--
+
+-- This SQL script renames all tables to lowercase.
+-- This is required for MariaDB on macOS or Windows because these
+-- operating systems use case insensitive filesystems.
+
+-- Table renames on such filesystems must use an intermediate table name
+-- because a direct `RENAME TABLE A TO a;` does not work.
+
+-- First, create a procedure to handle the renames
+DELIMITER //
+
+CREATE PROCEDURE rename_tables_to_lowercase()
+BEGIN
+    DECLARE done INT DEFAULT FALSE;
+    DECLARE table_name_var VARCHAR(255);
+    
+    -- Cursor for all tables with uppercase characters
+    DECLARE table_cursor CURSOR FOR 
+        SELECT TABLE_NAME
+        FROM INFORMATION_SCHEMA.TABLES 
+        WHERE TABLE_SCHEMA = DATABASE()
+        AND TABLE_NAME != LOWER(TABLE_NAME);
```

Comment:
Hier kann man – wenn dies tatsächlich erforderlich ist – alternativ oder als weitere Bedingung ergänzen, dass nur ausgewählte Tabellen umbenannt werden: `AND TABLE_NAME IN ('Tabelle1', 'Tabelle2', ...)` (siehe Diskussionsbeitrag, wo angemerkt wird, dass die Kitodo-Datenbank auch Tabellen enthalten kann, die nicht zu Kitodo gehören).